schematics/playground-module/generate-missing-modules.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { basename, dirname, normalize, Path } from '@angular-devkit/core';
+import { apply, chain, DirEntry, filter, mergeWith, Rule, template, Tree, url } from '@angular-devkit/schematics';
+import {
+  generateCurrentDirImport,
+  generateFeatureModuleClassName,
+  generateFeatureModuleFileName,
+  generateRoutingModuleClassName,
+  generateRoutingModuleFileName,
+  getModuleDirs,
+  getPlaygroundRootDir,
+  hasComponentsInDir,
+  hasFeatureModuleInDir,
+  hasRoutingModuleInDir,
+  isFeatureModule,
+} from '../utils';
+
+export function generateMissingModules(tree: Tree): Rule {
+  const moduleDirs = [ getPlaygroundRootDir(tree), ...getModuleDirs(tree) ];
+
+  return chain(moduleDirs.map(moduleDir => fromTemplate(tree, optionsFromDir(moduleDir))));
+}
+
+function fromTemplate(tree: Tree, options: Object): Rule {
+  const transformedSource = apply(
+    url('./files'),
+    [
+      template(options),
+      filter(shouldCreateModule.bind(null, tree)),
+    ],
+  );
+
+  return mergeWith(transformedSource);
+}
+
+function shouldCreateModule(tree: Tree, filePath: Path): boolean {
+  const dir = tree.getDir(normalize(dirname(filePath)));
+  const fileName = basename(filePath);
+  const isModuleExist = isFeatureModule(fileName)
+    ? hasFeatureModuleInDir(dir)
+    : hasRoutingModuleInDir(dir);
+
+  if (isModuleExist) {
+    return false;
+  }
+
+  return isFeatureModule(fileName)
```

Comment:
I think it would much cleaner if we rewrite this statement with plain if:
```
if (isFeatureModule(filename)) {
  return true;
}

return hasComponentsInDir(dir);
```
or
```
return isFeatureModule(filename) || hasComponentsInDir(dir);
```

I persannaly prefer the second one. What do you think?